packages/typescript-private/console-host-agent/src/http-specs/private-device.ts

```diff
@@ -110,6 +116,9 @@ export class UpdateDeviceRequestBody
   @Type(() => Number)
   resolutionHeight!: number;
 
+  @IsString()
+  memory!: string;
```

Comment:
Does it need nullable or `IsNotEmpty()` here?

Reply:
@Ikalli And it hasn't been verified that there's a value for the emulator or every case. In this case, I didn't want the device creation to fail on the console. <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Smilies/Face%20Screaming%20in%20Fear.png" alt="Face Screaming in Fear" width="25" height="25" />